x-pack/plugins/observability/public/components/slo/slo_outdated_callout/index.tsx

```diff
@@ -25,37 +25,40 @@ export function SloOutdatedCallout() {
   const { isLoading, data } = useFetchSloDefinitions({ includeOutdatedOnly: true });
   if (!isLoading && data && data.total > 0) {
     return (
-      <EuiCallOut
-        color="warning"
-        iconType="warning"
-        title={i18n.translate('xpack.observability.slo.outdatedSloCallout.title', {
-          defaultMessage: '{total} Outdated SLOs Detected',
-          values: {
-            total: data.total,
-          },
-        })}
-      >
-        <p>
-          <FormattedMessage
-            id="xpack.observability.slo.outdatedSloCallout.message"
-            defaultMessage="We've noticed that you have {total} outdated SLO definitions, these SLOs will not be running or alerting until you've reset them. Please click the button below to review the SLO definitions; you can choose to either reset the SLO definition or remove it."
-            values={{ total: data.total }}
-          />
-        </p>
-        <p>
-          <EuiButton
-            color="warning"
-            data-test-subj="o11ySloOutdatedCalloutViewOutdatedSloDefinitionsButton"
-            fill
-            onClick={handleClick}
-          >
+      <>
```

Comment:
@shahzad31 Do I need to check anything here? As far as I can tell, you added a spacer. Anything else I am missing?